.travis.yml

```diff
@@ -4,8 +4,8 @@ env:
   global:
     - FORCE_COLOR=true
   matrix:
-    - ESLINT=5
-    - ESLINT=6
+    - ESLINT=7.5
```

Comment:
Why is this back again to just testing ESLint v7.X? As previously discussed, this should maintain compatibility with ESLint >= 5 so we need v5, v6 and v7 in the matrix.

Reply:
I did this on purpose, since `@typescript-eslint/experimental-utils@4.x` requires `eslint@^7.5.0` in order to have the optional chaining support